src/status_im/chat/handlers.cljs

```diff
@@ -291,18 +289,18 @@
 (register-handler :add-chat-loaded-callback
   (fn [db [_ chat-id callback]]
     (log/debug "Add chat loaded callback: " chat-id callback)
-    (update-in db [::chat-loaded-callbacks chat-id] conj callback)))
+    (update-in db [:chat-loaded-callbacks chat-id] conj callback)))
```

Comment:
i've removed all namespaced, i don't think it's a good practive to use namespaced